src/pages/index.page.jsx

```diff
@@ -17,7 +17,7 @@ export default function Home() {
                 we&#39;re adding new components frequently!
               </p>
             </div>
-            <div>
+            <div className="home-top__image-column">
```

Comment:
Looks like the scss for this class got commented out or isn't in use. Just wanted to bring this to your attention if you delete those.

Reply:
Right, it was not being used anywhere on the site so I removed it. Monet agreed it was minor/in scope so I went for it and cleaned it up!

Reply:
Looks like this got added back in on accident. Did you still want it to be removed?